internal/controller/kyma_controller.go

```diff
@@ -426,35 +411,28 @@ func (r *KymaReconciler) handleDeletingState(ctx context.Context, kyma *v1beta2.
 
 	if r.WatcherEnabled(kyma) {
 		if err := r.SKRWebhookManager.Remove(ctx, kyma); err != nil {
-			// error is expected, try again
-			r.enqueueNormalEvent(kyma, webhookChartRemoval, err.Error())
-			return ctrl.Result{RequeueAfter: r.RequeueIntervals.Busy}, nil
```

Comment:
Here the busy interval requeue gets replaced with immediate requeue. What's the reasoning behind?

Reply:
to compliance with requeue standard, when err occurs, requeue with error, there is no needs to control requeue interval for error, it's controlled by rate limit.